force-app/main/default/classes/SOQL.cls

```diff
@@ -51,8 +56,20 @@ public with sharing class SOQL {
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
     //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<ID> getContactIdsByLastName() {
+        // Query 
+        List<Contact> contacts = [SELECT Id FROM Contact WHERE LastName IN ('Smith', 'Jones')];
```

Comment:
You can also try using a bind variable to a Set<String> with the IN operator like so:
`Set<String> lastNames = new Set<String>{'a', 'b'};
`WHERE LastName IN :lastNames`